manipulation/models/pr2_collision_convex/base_v0/base_L.obj

```diff
@@ -0,0 +1,108 @@
+####
+#
+# OBJ File Generated by Meshlab
```

Comment:
why do we need the convex meshes?  if you just mark the file convex, doesn't drake do that for you?

Reply:
Switched back to just using the visual geometry as collision geometry, as in the original drake model.